src/expandCollapseUtilities.js

```diff
@@ -624,7 +630,11 @@ return {
     
     return current;
   },
-  repairEdges: function(node) {
+  repairEdge: function (edge) {
+    this.repairEdgesForNode(edge.source());
+    this.repairEdgesForNode(edge.target());
+  },
+  repairEdgesForNode: function(node) {
```

Comment:
Added a "repairEdge" function that repairs meta links for edges after they have been expanded